File: backend/app/schemas/branding.py
from datetime import datetime
from uuid import UUID

from pydantic import BaseModel, ConfigDict


class BrandingProfileResponse(BaseModel):
    model_config = ConfigDict(from_attributes=True)

    id: UUID
    organization_id: UUID
    name: str
    company_name: str
    accent_color: str
    logo_text: str | None
    contact_email: str | None
    cover_tagline: str | None
    footer_note: str | None
    is_default: bool
    created_at: datetime

File: frontend/types/branding.ts
export type BrandingProfile = {
  id: string;
  organization_id: string;
  name: string;
  company_name: string;
  accent_color: string;
  logo_text: string | null;
  contact_email: string | null;
  cover_tagline: string | null;
  footer_note: string | null;
  is_default: boolean;
  created_at: string;
};
